authorization-for-rag/langchain-next-js/src/lib/actions/documents.ts

```diff
@@ -37,7 +37,7 @@ export const createDocument = async (input: NewDocumentParams, text: string) =>
     );
 
     // write the relationship tuples to FGA
-    await addRelation(user.email, document.id);
+    await addRelation(user.sub, document.id);
```

Comment:
all `user.sub` references can be reverted in this file (please just inspect diff here when applying)